packages/x-date-pickers/src/internals/hooks/useField/useField.utils.ts

```diff
@@ -899,7 +899,11 @@ export const getSectionOrder = (
   while (RTLIndex >= 0) {
     groupedSectionsEnd = sections.findIndex(
       // eslint-disable-next-line @typescript-eslint/no-loop-func
-      (section, index) => index >= groupedSectionsStart && section.endSeparator?.includes(' '),
+      (section, index) =>
+        index >= groupedSectionsStart &&
+        section.endSeparator?.includes(' ') &&
+        // Special case where the spaces were nto there in the initial input
+        section.endSeparator !== ' / ',
```

Comment:
@flaviendelangle FYI, this might need adjustment depending on the approach we select for the format issue. 😉